content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+@cornish_systems_2013 encountered similar results for sequences of categorical items that had no inherent meaning:
+since participants must reproduce a whole set of sequences, or acoustic signals, at each generation, the set behaves as a interconnected system for which learning pressures gradually increase the combinatorial structure.
+
+Experimental semiotics and iterated learning also cross-fertilise each other:
+after @garrod_foundations_2007 used a Pictionary-like collaborative task, without transmission, to study the emergence of symbolism in a lexicon, a process they called "interactive grounding", @fay_interactive_2010 extended the findings to a micro-society.
+Fay and colleagues showed that globally shared symbols can emerge through the gradual alignment of such interactive groundings, leading to an increasingly refined and streamlined symbol system.
+The authors thus proposed "symbolisation" as an additional mechanism in the emergence of communication systems, based on intra-generational collaborative coordination through interaction, and parallel to the inter-generational learning biases and bottlenecks studied by iterated learning studies.
+Taking another leaf from experimental semiotics, @winters_languages_2015 exploited the observation that a large part of the meaning of an utterance comes from its situational context:
+they studied the influence of the situations in which participants use vocabulary items on the structure of the language that evolves from interaction and iterated learning [thus extending @silvey_word_2015].
+The authors show that, if the situations in which participants communicate do not contrast items on all the dimensions on which they differ, then the vocabulary set often evolves to not encode those dimensions.
+In other words, if the usage situations shield the participants from certain contrasts between items, the final vocabulary is often under-specified with respect to the full item space:
+it does not encode the dimensions that discriminate the unobserved contrasts, instead adapting to be useful only for the contrasts that users observed.
+This stream of research is active and gradually relaxing the constraining hypotheses made by initial studies.
+@carr_cultural_2017, for instance, recently made the space of items more realistic by exploring the emergence of vocabulary sets where participants communicate about a continuous unbounded set of items.
+
+A second, closely related question concerns how an already structured communication system evolves given a set of external, learning, or interaction pressures.
+@croft_evolutionary_2013 provides a general framework for this question, inspired by principles from biological evolution and by recent debates on the nature of the evolutionary process, and crucially focused on identifying an adequate unit of analysis for the evolutionary study of language change.
```

Comment:
### Camille
le paragraphe sur Croft, là, ça me semble trop obscur / obfuscated.  À la fois parce qu'il y a beaucoup d'implicite, et parce qu'on ne comprend pas bien quels sont les débats sous-jacents, ni quel est l'intérêt premier de mentionner ce travail: est-ce parce qu'il y a une population of utterances ? (auquel cas ça se relie à une notion de la CAT), que veut dire / à quoi sert "On the other side, he draws on the critique that Developmental Systems Theory opposes to a gene-centred view of biological evolution (Oyama, Griffiths, and Gray 2001), and insists that a theory of language change should consider utterances to be full life cycles made of pronunciation, meaning, and interpretation in context" ?  est-ce que les "linguemes" sont une construction de Croft ou bien est-ce qu'il en fait quelque chose de spécial pour proposer sa TUS ? en quoi cette théorie évolutionnaire très intégrée se connecte au reste du SoA de la section ? est-ce que cela ne requiert pas une sous-section / un marqueur de \paragraph{} en propre ?

Reply:
Mostly removed Croft. Done.